src/quo/components/wallet/progress_bar/style.cljs

```diff
@@ -7,27 +7,41 @@
   {:light {:pending   {:border-color     colors/neutral-80-opa-5
                        :background-color colors/neutral-5}
            :confirmed {:border-color     colors/neutral-80-opa-5
-                       :background-color (colors/custom-color :success 50)}
+                       :background-color (colors/resolve-color :success 50)}
            :finalized {:border-color     colors/neutral-80-opa-5
-                       :background-color (colors/custom-color customization-color 50)}
+                       :background-color (colors/resolve-color customization-color 50)}
            :error     {:border-color     colors/neutral-80-opa-5
-                       :background-color (colors/custom-color :danger 50)}}
+                       :background-color (colors/resolve-color :danger 50)}}
    :dark  {:pending   {:border-color     colors/neutral-70
                        :background-color colors/neutral-80}
            :confirmed {:border-color     colors/white-opa-5
-                       :background-color (colors/custom-color :success 60)}
+                       :background-color (colors/resolve-color :success 60)}
            :finalized {:border-color     colors/neutral-80-opa-5
-                       :background-color (colors/custom-color customization-color 60)}
+                       :background-color (colors/resolve-color customization-color 60)}
            :error     {:border-color     colors/white-opa-5
-                       :background-color (colors/custom-color :danger 60)}}})
+                       :background-color (colors/resolve-color :danger 60)}}})
 
 (defn root-container
-  [{:keys [customization-color state theme]}]
+  [{:keys [customization-color state theme full-width?]}]
   (let [{:keys [background-color border-color]} (get-in (border-and-background-color customization-color)
-                                                        [theme state])]
+                                                        [theme (if full-width? :pending state)])]
+    {:height            12
+     :flex              (when full-width? 1)
+     :width             (when (not full-width?) 8)
+     :border-radius     3
+     :border-width      1
+     :border-color      border-color
+     :background-color  background-color
+     :margin-horizontal 1
+     :margin-vertical   2}))
+
+(defn progressed-bar
+  [{:keys [customization-color state theme progressed-value]}]
+  (let [{:keys [background-color]} (get-in (border-and-background-color customization-color)
+                                           [theme state])
+        progress                   (if (> progressed-value 100) 100 progressed-value)]
```

Comment:
should we put `100` in a constant? e.g `max-value` or something like that? 🤔

Reply:
I don't think so. The max value is always 100%. So there is no need to do that.

Reply:
Done